Basic Note Application/Scenes/Auth/Register/RegisterViewController.swift

```diff
@@ -9,19 +9,28 @@ import UIKit
 import TinyConstraints
 import Alamofire
 
-class RegisterViewController: UIViewController {
+class RegisterViewController: BaseViewController<RegisterViewModel> {
     
     private let primaryButton = PrimaryButton()
     private let authTextField = AuthTextField()
     private let authTextField2 = AuthTextField()
     private let authTextField3 = AuthTextField()
     private let authSignUpView = AuthSignUpView()
     
-    private let stackView: UIStackView = {
+    private let scrollView: UIScrollView = {
```

Comment:
```suggestion
    private let scrollView = UIScrollView()
```
parametre setlenmediğinden kısaca böyle yazılabilir